### PR TITLE
General improvements in quickstart.md

### PR DESCRIPTION
**Pull request by system32uwu**
Capitalization, grammar, punctuation, references, clarifications.

### CODE REVIEW

File: docs/content/quickstart.md [hunk 1]
```diff
@@ -4,51 +4,51 @@ description: "start here!"
...
 
 ```cs
 context.Branch = Branch.None;
 ```
 
-# configuring
+# Configuring
 
-by default, workspacer loads a default configuration, but you will likely want to customize this in order to do things like modify the widgets on the bar, add workspaces, switch workspace containers, or otherwise change the default behavior of workspacer.
+Workspacer loads a default configuration with almost everything you'd need, but you will likely want to modify things like [key bindings](#how-do-i-register-custom-keybindings), [widgets](#menu-bar-widgets) on the bar, [workspaces](#how-do-i-define-custom-workspaces), [layouts](#layout-engines), [plugins](#plugins) or otherwise the default behavior of workspacer.
```
Comment (system32uwu) [inline docs/content/quickstart.md:32]: How can I reference this headers that are in other pages? (If there's any way to do it without typing the full url)

Comment (system32uwu) [reply docs/content/quickstart.md:32]: #/config#how-do-i-register-custom-keybindings maybe?

Comment (system32uwu) [reply docs/content/quickstart.md]: Yup that worked

File: docs/content/quickstart.md [hunk 2]
```diff
@@ -4,51 +4,51 @@ description: "start here!"
 type: docs
 ---
 
-# installing
+# Installing
```
Comment (dalyIsaac) [inline docs/content/quickstart.md:7]:
```suggestion
## Installing
```

Using h2 headings is consistent with `config.md`

File: docs/content/quickstart.md [hunk 3]
```diff
@@ -4,51 +4,51 @@ description: "start here!"
...
 
-download the installer or the zip file from the [Releases](https://github.com/workspacer/workspacer/releases/latest) page and install/unzip it anywhere you like. 
+Download the installer or the zip file from the [Releases](https://github.com/workspacer/workspacer/releases/latest) page and install/unzip it anywhere you like. 
 
-alternatively, workspacer is available via [Chocolatey](https://community.chocolatey.org/packages/workspacer):
+Alternatively, workspacer is available via [Chocolatey](https://community.chocolatey.org/packages/workspacer):
 
 ```
```
Comment (dalyIsaac) [inline docs/content/quickstart.md:13]:
```suggestion
```shell
```

File: docs/content/quickstart.md [hunk 4]
```diff
@@ -4,51 +4,51 @@ description: "start here!"
...
-and [Scoop](https://scoop.sh/):
+and via [Scoop](https://scoop.sh/):
 
 ```
 scoop bucket add extras
 scoop install workspacer
 ```
 
-workspacer automatically checks for updates. however, if you are using a package manager with either Chocolatey or Scoop, you should uncomment/add this line in your workspacer config to disable autoupdates:
+Workspacer automatically checks for updates. However, if you are using a package manager (either Chocolatey or Scoop), you should uncomment/add this line in your workspacer configuration file to disable autoupdates:
```
Comment (dalyIsaac) [inline docs/content/quickstart.md:24]:
```suggestion
Workspacer automatically checks for updates. However, if you are using a package manager (either Chocolatey or Scoop), you should uncomment/add this line in your workspacer configuration file to disable automatic updates:
```

File: docs/content/quickstart.md [hunk 5]
```diff
@@ -4,51 +4,51 @@ description: "start here!"
...
 
 ```cs
 context.Branch = Branch.None;
 ```
 
-# configuring
+# Configuring
```
Comment (dalyIsaac) [inline docs/content/quickstart.md:30]:
```suggestion
## Configuring
```

File: docs/content/quickstart.md [hunk 6]
```diff
@@ -4,51 +4,51 @@ description: "start here!"
...
+Workspacer's configuration file allows for __type checking__ and __intellisense__! currently, only VSCode with the C# extension is supported. Opening the `.workspacer` folder in your home directory inside VSCode with the C# extension installed will provide syntax highlighting, type checking, and autocompletion automatically because of the [`#r` directives at the top of the config file](https://github.com/workspacer/workspacer/blob/master/src/workspacer/Config/workspacer.config.template.csx) (so don't remove those, or else this won't work!).
 
-[the config page](/config) lists a few examples and tips/tricks on how to customize your configuration to your liking.
+[The config page](/config) lists a few examples and tips & tricks on how to customize workspacer to your liking.
 
-# learning
+# Learning
```
Comment (dalyIsaac) [inline docs/content/quickstart.md:46]:
```suggestion
## Learning
```

File: docs/content/quickstart.md [hunk 7]
```diff
@@ -4,51 +4,51 @@ description: "start here!"
...
 
-workspacer's default configuration behaves like other tiling window managers, but deviates where not possible due to limitations in `Win32` that prevent controlling windows in the same way a tiling window manager would be able to when controlling the actual management of windows like an X11 based window manager.
+Workspacer's default configuration makes it behave like other tiling window managers, but deviates where not possible due to limitations of the [`Win32` API](https://docs.microsoft.com/en-us/windows/win32/api/) that prevents from freely controlling windows in the same way as an X11 tiling window manager.
 
-[the keybindings page](/keybindings) describes the default set of keybindings.
+[The keybindings page](/keybindings) shows the default set of keybindings.
 
-# submit feedback!
+# Submit feedback!
```
Comment (dalyIsaac) [inline docs/content/quickstart.md:52]:
```suggestion
## Submit feedback!
```

File: docs/content/quickstart.md [hunk 8]
```diff
@@ -4,51 +4,51 @@ description: "start here!"
...
+Alternatively, workspacer is available via [Chocolatey](https://community.chocolatey.org/packages/workspacer):
 
 ```
 choco install workspacer
 ```
 
-and [Scoop](https://scoop.sh/):
+and via [Scoop](https://scoop.sh/):
 
 ```
```
Comment (dalyIsaac) [inline docs/content/quickstart.md:19]:
```suggestion
```shell
```

**Review by dalyIsaac — COMMENTED** [review]
Added some suggestions for formatting and the odd grammatical improvement

**Review by dalyIsaac — APPROVED** [review]
LGTM!

**Review by N1x0 — APPROVED** [review]

**Pull request merged**